src/args.rs

```diff
@@ -83,6 +83,10 @@ pub struct CliArgs {
     #[clap(short = 'H', long = "hidden")]
     pub hidden: bool,
 
+    /// Enable change theme display
+    #[clap(short = 'c', long = "change-theme")]
```

Comment:
Also this can't work since now you use `-c` twice. Does this need a short option?

Reply:
Oh,I didn't notice it was used,I'll remove it.